compiler/src/main/java/io/grpc/kotlin/generator/protoc/ProtoMethodName.kt

```diff
@@ -23,7 +23,7 @@ import com.google.protobuf.Descriptors.MethodDescriptor
 /** Represents the unqualified name of an RPC method in a proto file, in UpperCamelCase. */
 data class ProtoMethodName(val name: String) : CharSequence by name {
   fun toMemberSimpleName(): MemberSimpleName =
-    MemberSimpleName(CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_CAMEL, name))
+    MemberSimpleName(CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_CAMEL, name)).handleSpecialCharacters()
```

Comment:
This honestly looks like the processing ought to happen *here,* not in MemberSimpleName.

Reply:
Moved code from MemberSimpleName and split into two methods